Nighttime-Operations/Auxiliary-Telescope/AT-On-Sky/CWFS.rst

```diff
@@ -4,11 +4,92 @@
+
+Overview
+========
+
+This procedure must be followed by the observers before the beginning of dark time, prior to starting observations. 
+It will focus the telescope, center a target and absorb the pointing offsets. 
+
+.. _Beginning_of_the_night-Prerequisites:
+
+Prerequisites
+=============
+- The telescope is ready for on-sky operations: it has been vented (if allowed), mirror cover and dome shutter are open, AOS corrections are enabled. (:ref:`Prepare for on-sky <AT-Pre-Open-for-On-Sky-Operations>`)
+- The sky brightness is low enough that a CWFS can converge succesfully. This condition is normally achieved around 10-15 minutes before the end of nautical twilight or dark time. 
+
+.. _Beginning_of_the_night-Post-Conditions:
+
+Post-Condition
+===============
+
+- Telescope is focused and pointing offsets have been absorbed. The system is ready to start scheduler-driven observations (:ref:`Scheduler <Nighttime-Scheduler-Scheduler-Operational-Procedures>`) or engineering tasks. 
+
+.. _Beginning_of_the_night-Procedure-Steps:
+
+Procedure Steps
+===============
+
+- Confirm sky background is dark enough. 
+
+As a general rule, a good time to start executing the following SAL scripts is 10-15 minutes before the beginning of dark time. 
+
+- Launch a CWFS sequence on an 8th magnitude star. 
+
+From the LOVE ``ATQueue`` panel, under ``AVAILABLE SCRIPTS`` on the left side, find and load the script ``auxtel/latiss_cwfs_align.py`` by clicking on the blue icon. 
+It is included under ``External scripts``. 
+
+In the ``configuring Script: latiss_cwfs_align`` window, the script configuration is introduced under ``CONFIG``.
+Traditionally, we perform the first CWFS of the night towards the south and at mid-elevations on an 8-mag star. 
+With the configuration below, the script will query SIMBAD for a target around the provided ``az`` and ``el`` coordinates and with the specified maximum brightness ``mag_limit``.
+
+.. code-block:: text
+    
+    find_target:
+        az: 180
+        el: 60
+        mag_limit: 8.0
+    
+Click on ``Add`` and the script will be added to the end of the script queue.
+
+.. figure:: ./_static/CWFS_AuxTel.png
+    :name: latiss-cwfs
+
+    Screenshot of LOVE launching a CWFS. 
+
+- Instruct the telescope to correct the pointing, if needed. 
+
+This SAL script will measure and apply the zero point correction to the telescope pointing. 
+The steps the script perform in the background are:
+    - Reset the pointing and hexapod x and y offsets.
+    - Acquire target. How to select the source is summarize below. 
+    - Find the offset between the brightest source in the field and the boresight.
+    - Offset the telescope to place the source at the center of the detector. 
+    - The telescope will absorb the measured pointing offsets. 
+
+Nominally, the same target operated in the CWFS is used.  
+Add the external script ``auxtel/correct_pointing.py`` to the ``AT Queue`` with an empty configuration cell. 
```

Comment:
So @tribeiro and I disagree on this :) but you can add an `az` and `el` keyword to specify where to get a target.
I think we should automatically using the target that it's on, as the default case for this script will be at the beginning of the night.
Option #2 would be to grab the current telescope position and use that by default.
I think for the moment, I'd suggest adding the az and el parameters to the config, and making them the same as the CWFS above.

Reply:
Hoping both are happy this way